packages/dd-trace/src/appsec/iast/analyzers/vulnerability-analyzer.js

```diff
@@ -43,6 +58,10 @@ class Analyzer extends Plugin {
     }
   }
 
+  analyzeAll (...values) {
+    values.forEach(value => this.analyze(value))
```

Comment:
It is not optimal, because each `this.analyze` tries to get store, context etc. It think that it will be better if we implement custom `analyze` in `ldapjs-injection-analyzer` (like in path-traversal-analyzer) or if we create other `analyzeAll` method getting the context only once.